### ISSUE TITLE
Use of install.libs.R

### ISSUE THREAD

**Issue by jtbates** [hadley/devtools#1444]
I am working on a package which builds a command line executable in src/ and then copies it to bin/  in the package directory using `install.libs.R` as suggested by the [Writing R Extensions manual](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-subdirectories). However, it appears that `devtools::load_all` does not support `install.libs.R`. For this reason I've had to turn off the option 'Use devtools package functions if available' in RStudio while developing the package. Is the lack of support for `install.libs.R` considered a bug? If not, is there a recommended way to install binaries that is compatible with devtools?

**Comment by jimhester** [comment]
I don't think I would call it a bug, `install.libs.R` is a pretty esoteric convention, it looks like [only around 20 packages use it in all of CRAN](https://github.com/search?utf8=%E2%9C%93&q=filename%3A%22install.libs.R%22+file.copy+user%3Acran&type=Code&ref=searchresults).

We might accept a PR adding support assuming it could be done with a minimum of extra code...

**Comment by jtbates** [comment]
Ah, okay. I had assumed it was more common since I found it in the manual. I've figured out how to install the binary without `install.libs.R`. If the target for the executable in `Makevars` is set to `../inst/bin/execname` then it can be found by `system.file('bin', 'execname', package=pkg_name)` because the shim checks in `inst/`.

I don't think it would be much code to add support for `install.lib.R` in the way that I was using it. It looks like there was code for it at one point in [packman](https://github.com/hadley/packman/blob/e73c9634a46989d38a584523b9a6dc0ac69c67c7/old/cmd-install.r#L399). However, I am not sure if some packages' `install.libs.R` scripts depend on the installation directory `R_PACKAGE_DIR` being different from the source directory `R_PACKAGE_SOURCE`. That would complicate things.

Would the relevant code to look at for adding support for `install.libs.R` be found in `pkgload::load_all`?
